.circleci/config.yml

```diff
@@ -228,6 +253,21 @@ jobs:
       - run:
```

Comment:
HMRC is setup to use an RDS postgres database and the delete step will only need to run once on legacy or live
I'd keep it on `delete_uat_branch_live` and remove the `Delete UAT release` from the legacy step.

Reply:
Thanks. Good spot. I've removed the delete database step from the legacy `delete_uat_branch` job.